inc/namespace.php

```diff
@@ -900,10 +900,11 @@ function elasticpress_analyzer_language() : string {
 function elasticpress_mapping( array $mapping, ?string $index = null ) : array {
 	// Derive current mapping type.
 	$indexables = Indexables::factory()->get_all( null, true );
+	$indexables = array_merge( $indexables, Indexables::factory()->get_all( true, true ) );
```

Comment:
The list excluded `user` because the first parameter for `get_all()` determines whether to get indexables scoped to the site or to the whole network - the user indexable is network wide.